src/classes/BDI_SettingsUI_CTRL.cls

```diff
@@ -94,7 +94,7 @@ public with sharing class BDI_SettingsUI_CTRL {
 
                 // this UI is specific to help text settings in current state
```

Comment:
Lets update this comment since this is no longer specific to help text mapping.

Reply:
@kenlewis-sfdo I totally agree with writing some tests for it. And I'll update the comment.